cpp/test/Glacier2/override/Client.cpp

```diff
@@ -26,7 +26,6 @@ CallbackClient::run(int argc, char** argv)
 {
     auto properties = createTestProperties(argc, argv);
     properties->setProperty("Ice.Warn.Connections", "0");
-    properties->setProperty("Ice.ThreadPool.Client.Serialize", "1");
```

Comment:
Removing the Serialize in this test doesn't appear to break the test. What are they for?

Reply:
Seeing that the server and client exchange a token with the callback, it's most likely to ensure oneway calls ordering. The test should set `MaxDispatches=1` instead.

Reply:
Set MaxDispatches=1 on what?

The CallbackReceiverAdapter?